__tests__/app.test.js

```diff
@@ -39,5 +39,49 @@ describe('GET requests', () => {
                 expect(body.msg).toBe('Path not found!')
             })
         });
-    })
-})
\ No newline at end of file
+    describe('GET /api/reviews/:review_id', () => {
+        it('should respond with a status 200 and a review object', () => {
+            const reviewId = 10
+            return request(app)
+            .get(`/api/reviews/${reviewId}`)
+            .expect(200)
+            .then((res) => {                            
+            expect(res.body.reviews).toHaveProperty('review_id', expect.any(Number))
```

Comment:
The expected keys should have correct values. They must be values related to the requested review_id, e.g. review_id: 10